cosmos/x/evm/keeper/msg_server.go

```diff
@@ -71,7 +71,7 @@ func (k *Keeper) UpdateParams(
 	}
 
 	// Update the params.
-	k.cp.Prepare(ctx)
-	k.cp.SetParams(&req.Params)
+	k.Host.Cp.Prepare(ctx)
```

Comment:
Cp become public because of this usage

Reply:
I think it would be nice if we could keep it private